packages/tabs/partials/tab-triggers/index.tsx

```diff
@@ -61,7 +63,7 @@ function TabTriggers({
    */
   useEffect(() => {
     // Determine which arrows to show
-    const { scrollLeft, scrollWidth, offsetWidth } = scrollRef.current
+    const { scrollLeft, scrollWidth, offsetWidth } = scrollRef.current!
```

Comment:
This potentially feels like a scenario where we might want to check that `scrollRef.current` is defined, but as it was like this before I don't think it's critical.